src/KnowbaseItem.php

```diff
@@ -885,12 +756,11 @@ public function post_purgeItem()
         NotificationEvent::raiseEvent('delete', $this);
     }
 
-
     /**
      * Print out an HTML "<form>" for knowbase item
      *
-     * @param $ID
-     * @param $options array
+     * @param integer $ID
```

Comment:
That does not have much effect; PHP does not care about docs, and IDEs/phpstan works well with both notations

Reply:
It doesn't matter to me. The GLPI developer documentation specifically mentions integer and boolean rather than int and bool though. I thought it explicitly said that the longer name had to be used, but I don't see that now.

Reply:
I guess we already have enough real issue to solve :D